CHANGELOG.md

```diff
@@ -9,6 +9,33 @@ You can find and compare releases at the [GitHub release page](https://github.co
 
 ## Unreleased
 
+### Changed
+
+- New signature for `handleFieldBuilder` on `FieldBuilderDirective` interface
+
+```diff
+<?php
+
+namespace Nuwave\Lighthouse\Support\Contracts;
+
++ use GraphQL\Type\Definition\ResolveInfo;
```

Comment:
Missing import for `GraphQLContext`

Reply:
They're both in the same namespace though, is it necessary to add another use statement for it?

Reply:
I changed the example to be user-facing - it concerns them when they implement custom directives.